### PR TITLE
support japanese and russian translation J5

### PR DESCRIPTION
**Pull request by heelc29**
@tecpromotion The time slots must be defined. Suggestion?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,13 @@ These two files can't be processed manual, because of the amount of work and ris
...
 * UTC 04:01-05:00 -> J5 Download Package Translations Crowdin Action
 * UTC 07:12-08:12 -> Get J5 Core Source and Upload to Crowdin
-* UTC 08:01-08:15 -> Get Russian and upload Translations to Crowdin
+* UTC 08:01-08:15 -> Get J4 Russian and upload Translations to Crowdin
 * UTC 08:16-08:30 -> Get J5 Ukrainian and upload Translations to Crowdin
-* UTC 08:31-08:45 -> Get Japanese and upload Translations to Crowdin
+* UTC 08:31-08:45 -> Get J4 Japanese and upload Translations to Crowdin
 * UTC 08:46-09:00 -> Get J4 Ukrainian and upload Translations to Crowdin
 
+* UTC TBD -> Get J5 Russian and upload Translations to Crowdin
```
Comment (tecpromotion) [inline README.md:37]:
```suggestion
* UTC 09:01-09:15 -> Get J5 Russian and upload Translations to Crowdin
```

File: README.md [hunk 2]
```diff
@@ -29,11 +29,13 @@ These two files can't be processed manual, because of the amount of work and ris
...
+* UTC TBD -> Get J5 Japanese and upload Translations to Crowdin
```
Comment (tecpromotion) [inline README.md:38]:
```suggestion
* UTC 09:16-09:30 -> Get J5 Japanese and upload Translations to Crowdin
```

File: README.md [hunk 3]
```diff
@@ -29,11 +29,13 @@ These two files can't be processed manual, because of the amount of work and ris
 * UTC 03:30-04:30 -> J5 Download Installer Translations Crowdin Action
 * UTC 04:01-05:00 -> J5 Download Package Translations Crowdin Action
 * UTC 07:12-08:12 -> Get J5 Core Source and Upload to Crowdin
-* UTC 08:01-08:15 -> Get Russian and upload Translations to Crowdin
+* UTC 08:01-08:15 -> Get J4 Russian and upload Translations to Crowdin
 * UTC 08:16-08:30 -> Get J5 Ukrainian and upload Translations to Crowdin
-* UTC 08:31-08:45 -> Get Japanese and upload Translations to Crowdin
+* UTC 08:31-08:45 -> Get J4 Japanese and upload Translations to Crowdin
 * UTC 08:46-09:00 -> Get J4 Ukrainian and upload Translations to Crowdin
```
Comment (tecpromotion) [inline README.md:36]:
```suggestion
```

**Review by tecpromotion — CHANGES_REQUESTED** [review]
If nothing else is running, simply maintain the 15-minute cycle.

**Comment by heelc29** [comment]
> If nothing else is running, simply maintain the 15-minute cycle.

Done.

> UTC 09:00-10:00 -> Project Build J3

is no longer relevant?

**Comment by tecpromotion** [comment]
> > If nothing else is running, simply maintain the 15-minute cycle.
> 
> Done.
> 
> > UTC 09:00-10:00 -> Project Build J3
> 
> is no longer relevant?

no.

**Comment by tecpromotion** [comment]
> > > If nothing else is running, simply maintain the 15-minute cycle.
> > 
> > 
> > Done.
> > > UTC 09:00-10:00 -> Project Build J3
> > 
> > 
> > is no longer relevant?
> 
> no.

see https://github.com/joomla/core-translations/pull/837

**Review by tecpromotion — APPROVED** [review]
thanks @heelc29

**Pull request merged**